src/definitions/general_config.rs

```diff
@@ -2,7 +2,11 @@ use crate::{business_logic::state::state_api_objects::BlockInfo, utils::Address}
 use felt::Felt;
 use getset::{CopyGetters, MutGetters};
 use num_traits::Zero;
+use std::collections::HashMap;
 
+use super::error::StarknetChainIdError;
```

Comment:
There's no real problem here (yet), but I usually like grouping all imports in a single block. If only `rust-analyzer` modifies the imports, then it ***should*** be no problem, but when we add a missing import manually and place it in the wrong block then `rust-analyzer` gets confused and gets confusing to read. By having them all in a single block this can be avoided.

```suggestion
use std::collections::HashMap;
use super::error::StarknetChainIdError; // then format the document to order them correctly
```

Tip: Useful VSCode/VSCodium setting when using `rust-analyzer`: `"rust-analyzer.imports.group.enable": false`.

(if you decide to change it, know that this is not the only place)

Reply:
i will check Vs configuration, but this impot was deleted.